src/PresentationalComponents/Shield/Shield.js

```diff
@@ -0,0 +1,76 @@
+import { QuestionIcon, SecurityIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React from 'react';
+import { impactList, colorList } from './consts';
+
+class Shield extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.title = this.props.tooltipPrefix + (this.props.title || this.getColoredBadgeByImpact().title);
```

Comment:
Assigning to attributes is generally not a good idea. With rhis approach it means that whever consumer changes title it will not get re-rendered. Can you move this to render and getColoredBadgeByImpact please?